genie_annotations.json

```diff
@@ -0,0 +1,24 @@
+{
+  "tissueSource": [
```

Comment:
We'll need to remove sample-specific annotations from this dictionary for other reasons anyway.